prow/config/config.go

```diff
@@ -513,25 +561,29 @@ type Plank struct {
 	PodUnscheduledTimeout *metav1.Duration `json:"pod_unscheduled_timeout,omitempty"`
 
 	// DefaultDecorationConfigs holds the default decoration config for specific values.
-	// Each entry in the slice specifies Repo and Cluster regexp filter fields to
-	// match against jobs and a corresponding DecorationConfig. All entries that
-	// match a job are used. Later matching entries override the fields of earlier
-	// matching entries.
```

Comment:
nit: This documentation is pretty important because it explains the order of precedence of the defaults. Please do not remove it.

Reply:
These lines were duplicated in the original; if you look at the comment for `DefaultDecorationConfigEntries` it's still there. I just thought that we shouldn't duplicate comments so I cleaned this up but I can put it back if you want. WDYT?

Reply:
I guess it makes more sense to comment this block instead and remove the duplicated comment for `DefaultDecorationConfigEntries`, because ultimately the `DefaultDecorationConfigs` is what we're going to use downstream.